src/api/googlesheetApi.js

```diff
@@ -2,20 +2,22 @@ import Tabletop from 'tabletop';
 
 function normalizeHeaders(element) {
   element["name"] = element["name"];
-  element["tags"] = String(element["serviceprovided"]).split(", ");
+  if (typeof element["serviceprovided"] !== "undefined") {
```

Comment:
You can skip `typeof` with the more simple:
```js
if(element["serviceprovided"] == undefined) { ... }
```




but even more simple, since `undefined` is falsy, you can do :
```js
if(element["serviceprovided"]) {  ... }
```